### PR TITLE
Try to smooth out the sidebar animation.

### PR DESCRIPTION
**Pull request by jbethanco**
https://user-images.githubusercontent.com/76016626/103868250-b9461500-506c-11eb-940c-3474ec20d9a9.mp4

Adjust some button sizes.

This isn't a pull demand... just a pull-request. 
Feel free to trash it.
This makes the sidebar animation smooth with an offset move and without making the buttons disappear.
I also added some sidebar button padding to make them not touch the edges of the screen.   Also made them a little shorter.  

Won't merge without full UI team buy-in. Or if the button changes are too much... please snipe the offset code and delete the branch and PR.

### CODE REVIEW

File: PBLogging/Views/SideViews/SideView.swift [hunk 1]
```diff
@@ -56,38 +54,41 @@ struct SideView: View {
                             // and use it to set the current Form
                         })
                     }
-                    // Keep this last to push everything up.
```
Comment (Pete-Hoch) [inline PBLogging/Views/SideViews/SideView.swift:59]: Why delete the comment? Is it in error? Is it as obvious as 2 + 2 = 4?

File: PBLogging/Views/SideViews/SideView.swift [hunk 2]
```diff
@@ -56,38 +54,41 @@ struct SideView: View {
...
+                
+                Spacer()
             }
+            
         }
-        .frame(width: isCollapsed ? 40 : 200)
-        .clipped()
+        .frame(width: 240)
+        .offset(x: isCollapsed ? -100 : 0)
+        .frame(width: isCollapsed ? 40 : 240)
```
Comment (Pete-Hoch) [inline PBLogging/Views/SideViews/SideView.swift:65]: I was just thinking earlier that the sideview should be narrower to make more room for the form data.

Why are there two .frame() calls? Maybe some comments would help.

File: PBLogging/Views/SideViews/SideViewButton.swift [hunk 3]
```diff
@@ -20,12 +20,14 @@ struct SideViewButton: View {
         }) {
             HStack {
                 BoldText(text: text, size: 14, color: .pblSlate)
-                    .padding(.all)
+                    .padding(EdgeInsets(top: 10, leading: 8, bottom: 10, trailing: 8))
```
Comment (Pete-Hoch) [inline PBLogging/Views/SideViews/SideViewButton.swift:23]: I like this, but I don't think we need to widen the sidebar because of it.

**Review by Pete-Hoch — COMMENTED** [review]
Just some comments. I'll play with it in the morning.

File: PBLogging/Views/SideViews/SideView.swift (hunk 1 above)
Comment (jbethanco) [reply PBLogging/Views/SideViews/SideView.swift:59]: It felt obvious. But that's to me. I forget that my instinctual deletions get committed as well.

File: PBLogging/Views/SideViews/SideView.swift (hunk 2 above)
Comment (jbethanco) [reply PBLogging/Views/SideViews/SideView.swift:65]: I'm actually not sure. I was fighting something with the animation at the end when it finally worked. I was probably fighting that second frame call. Theoretically it shouldn't need any frame width animation.

File: PBLogging/Views/SideViews/SideViewButton.swift (hunk 3 above)
Comment (jbethanco) [reply PBLogging/Views/SideViews/SideViewButton.swift:23]: I think I was just fiddling with that. We've got the padding there and outside the button. Try removing the one from the button and see how it looks.

File: PBLogging/Views/SideViews/SideView.swift (hunk 1 above)
Comment (petermisik) [reply PBLogging/Views/SideViews/SideView.swift:59]: I know some people feel comments are clutter, but for those that have left projects for others, they're a source of documentation.   With swiftUI, we know that we need a spacer to push, but is leaving this comment terrible?

**Review by petermisik — COMMENTED** [review]
Looking good so far.

File: PBLogging/Views/SideViews/SideView.swift (hunk 1 above)
Comment (jbethanco) [reply PBLogging/Views/SideViews/SideView.swift:59]: If we all know the spacer pushes, then leaving a comment that says spacer pushes wouldn't be a good thing. 
I know SwiftUI is new though and some things aren't as obvious to some as they are to others. I'll try to be less generous with the backspace. :)

File: PBLogging/Views/SideViews/SideView.swift (hunk 2 above)
Comment (petermisik) [reply PBLogging/Views/SideViews/SideView.swift:65]: If you drop the top .frame, you'll lose the chevron.

**Pull request closed without merging**